### PR TITLE
Follow plane when changing missions in Animate3DGraphic

### PR DESCRIPTION
**Pull request by williambohrmann3**
# Description

The scene's camera controller is null after Don't Follow button is clicked, keeping the camera in place. When changing missions, the camera should follow the plane, so the controller should be set to the OrbitGeoElementController. 

## Type of change

- Bug fix

## Platforms tested on

- [x] WPF .NET 6
- [x] WPF Framework
- [x] WinUI
- [x] MAUI WinUI
- [x] MAUI Android
- [x] MAUI iOS
- [x] MAUI MacCatalyst

## Checklist

- [x] Self-review of changes
- [x] All changes work as expected on all affected platforms
- [x] There are no warnings related to changes
- [x] Code is commented and follows .NET conventions and standards
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] No unrelated changes have been made to any other code or project files

### CODE REVIEW

File: src/MAUI/Maui.Samples/Samples/GraphicsOverlay/Animate3DGraphic/Animate3DGraphic.xaml [hunk 1]
```diff
@@ -10,6 +10,7 @@
                 BackgroundColor="{AppThemeBinding Light=#dfdfdf,
                                                   Dark=#303030}"
                 HorizontalOptions="End"
+                WidthRequest="400"
```
Comment (williambohrmann3) [inline src/MAUI/Maui.Samples/Samples/GraphicsOverlay/Animate3DGraphic/Animate3DGraphic.xaml:13]: This fixes a MacCatalyst specific UI issue where the border overlaps the buttons

**Review by amweis226 — CHANGES_REQUESTED** [review]
All platforms: text in mission button is left-justified, would look better centered like the other controls in that panel. "GrandCanyon" option is one word and also gets cut off by the picker

Maui Android: borders overlap in vertical mode, could use some help in horizontal mode. Worth noting that performance is not good on my simulator. Unsure if this is the case for others.

The "Maui android" issues need attention, the "all platforms" ones are less important.

**Comment by williambohrmann3** [comment]
MAUI UI has been updated to account for horizontal mode. On all platforms, all controls are now accessible.

**Comment by amweis226** [comment]
The controls are working properly now but in horizontal mode they take up about 80% of the screen, only a small sliver shows the SceneView

**Review by amweis226 — APPROVED** [review]

**Review by duffh — APPROVED** [review]

**Pull request merged**